api/trace/api.go

```diff
@@ -206,7 +206,7 @@ func WithStartTime(t time.Time) SpanOption {
 // data about the span.
 func WithAttributes(attrs ...core.KeyValue) SpanOption {
```

Comment:
can you please update the doc to indicate that multiple call to WithAttribute will appends the attributes? or something along the line.

Reply:
updated :)